apis/r/R/SOMADataFrame.R

```diff
@@ -415,13 +415,16 @@ SOMADataFrame <- R6::R6Class(
     #' @param new_shape An integer, greater than or equal to 1 + the
     #' `soma_joinid` domain slot.
     #' @return No return value
-    resize_soma_joinid_shape = function(new_shape) {
+    tiledbsoma_resize_soma_joinid_shape = function(new_shape) {
 
       stopifnot("'new_shape' must be an integer" = rlang::is_integerish(new_shape, n = 1) ||
         (bit64::is.integer64(new_shape) && length(new_shape) == 1)
       )
+      function_name_for_messages <- sys.call(sys.parent(n=1)[[1]]
```

Comment:
This is incorrect
```suggestion
      function_name_for_messages <- as.character(sys.call())
```
`sys.parent(1L)` would be the parent of the method, not the method itself (eg. `R_GlobalEnv` or the testthat `eval` environment); `sys.call()` also returns a `call`, and subsetting a `call` does not always return a `character`

Note, this returns `<name-of-object>$<method_name>`; if you want just the method name, we would need
```R
sub('.*\\$', replacement = '', x = as.character(sys.call()))
```